src/markdown/blueprints/resources/gs-assets.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: 'GS Assets'
```

Comment:
@rachelcyuan  @ryconn888 @abelhancock @jonmwood thoughts on this page name? is it clear enough?

Reply:
i'm merging this — but please, put your thoughts, we can always adjust later :)